src/components/todo-item/Index.module.css

```diff
@@ -0,0 +1,79 @@
+.container {
+  padding: 8px, 16px, 8px, 16px;
+  gap: 12px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+  width: 440px;
```

Comment:
Don't set a width. The element should grow according to it's parent. Now there is a gap on the right side. Check screenshot:

![CleanShot 2024-02-28 at 09 40 10@2x](https://github.com/modous/Todo-challenge/assets/148754291/2ccdb5e9-320d-43af-845a-b5ce7b20cef0)